cou/apps/channel_based.py

```diff
@@ -38,6 +38,24 @@ def get_latest_os_version(self, unit: Unit) -> OpenStackRelease:
         """
         return self.channel_codename
 
+    def expected_current_channel(self, target: OpenStackRelease) -> str:
```

Comment:
In base.py and auxiliary.py you used `_target` and here it's `target`. This needs to be unified.

Reply:
It's working, but it's strange :D. Now I understand how it works, so it's fine.